src/Auth/IamAuth.php

```diff
@@ -57,4 +57,12 @@ public function requestToken($request_data)
             throw new Exception('Failed to obtain new iamToken: response status is ' . $status);
         }
     }
+
+    public function decodeHeaderAndPayload(string $jwtToken) : array {
```

Comment:
need to move the function to StaticAuthentication and make private please, because it is not related to IamAuth